src/GT.UI/Views/Inquiries/_InquiriesOverview.cshtml

```diff
@@ -0,0 +1,24 @@
+﻿@inject SignInManager<ApplicationUser> SignInManager
```

Comment:
What is the reason for prefixing the cshtml with an underscore? (_)

Reply:
MVC Naming convention for Partials